src/python/WMQuality/WebTools/RESTClientAPI.py

```diff
@@ -1,13 +1,13 @@
+from future import standard_library
+standard_library.install_aliases()
+
 import hashlib
 import hmac
-import urllib
-from httplib import HTTPConnection
-
-try:
-    from urlparse import urlparse
-except ImportError:
-    # PY3
-    from urllib.parse import urlparse
+import urllib.parse
+
+from http.client import HTTPConnection
+from urllib.parse import urlparse
```

Comment:
How about removing this line and updating the line which uses urlparse to `urllib.parse.urlparse...`?

Reply:
Definitely tidier this way, thanks!